tools/lambda-promtail/lambda-promtail/main.go

```diff
@@ -25,19 +26,22 @@ const (
 
 	maxErrMsgLen = 1024
 
-	invalidExtraLabelsError = "invalid value for environment variable EXTRA_LABELS. Expected a comma separated list with an even number of entries. "
+	invalidExtraLabelsError  = "invalid value for environment variable EXTRA_LABELS. Expected a comma separated list with an even number of entries. "
+	invalidExtraHeadersError = "invalid value for environment variable EXTRA_HTTP_HEADERS. Expected a comma separated list with an even number of entries."
```

Comment:
depending on what we end up for the regex we're choosing to enforce, we should include the regex (or link to where the regex is defined, in the case of either of the 3 links you gave)

Reply:
I split the error messages and included the actual regex in the corresponding error message.